onnxruntime/core/optimizer/gelu_fusion.cc

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/optimizer/initializer.h"
+#include "core/optimizer/gelu_fusion.h"
+#include "core/graph/graph_utils.h"
+#include "float.h"
+#include <deque>
+
+using namespace ONNX_NAMESPACE;
+using namespace ::onnxruntime::common;
+namespace onnxruntime {
+
+static bool CheckConstantInput(const Graph& graph, const NodeArg& input_arg, float expected_value) {
+  auto shape = input_arg.Shape();
+  auto dim_size = shape->dim_size();
+  if (dim_size != 0) {
+    // only check scalar.
+    return false;
+  }
+
+  const ONNX_NAMESPACE::TensorProto* tensor_proto = nullptr;
+  graph.GetInitializedTensor(input_arg.Name(), tensor_proto);
+  auto init_const = std::make_unique<Initializer>(tensor_proto);
```

Comment:
Use graph_utils::GetConstantInitializer here instead, and remove the call to NodeArgIsConstant at the end as it isn't necessary if GetConstantInitializer returns a non-nullptr

Reply:
Fair. Thanks for the comment.,